Gir.toml

```diff
@@ -214,6 +214,144 @@ generate = [
     "Gtk.WrapMode",
 ]
 
+builders = [
+    "Gtk.AboutDialogBuilder",
```

Comment:
Hmm why do we not refer to the actual type name here but the Builder type? Will we also get `Gtk.BuilderBuilder`? ;)

It seems nicer to specify the actual types instead of the Builder type, that's what we do everywhere else in the configuration too.

Reply:
Gir support both way, for `builders` IMHO having suffix is more reasonable.
Although current version will have problem with `Gtk.Builder` instead `Gtk.BuilderBuilder`,
but it have only one `translation_domain: Option<String>`, so I don't generate it anyway.